src/app/BE/card/templates/Card.js

```diff
@@ -94,6 +114,78 @@ const DefaultCard = ({ parentCode, actions = [], code, columns }) => {
         {userType === 'HOST_CPY_REP' && <HCRDetail code={code} parentCode={parentCode} />}
       </Card>
     </MotionBox>
+  ) : (
```

Comment:
extract the logic to it's own file or make it generic not based on realm.

Reply:
Made a separate file for MM Agent